### PR TITLE
Update fh-mbaas-api to version 7.0.12

### PR DESCRIPTION
**Pull request by rachael-oregan**

### CODE REVIEW

**Review by pb82 — APPROVED** [review]
LGTM

File: test/unit/test-static-route.js [hunk 1]
```diff
@@ -1,14 +1,67 @@
...
+      },
+      mbaas: function (req, res, next) {
+        return next();
+      },
+      fhmiddleware: function () {
+        return function (req, res, next) {
+          return next();
+        };
+      },
+      cloud: function () {
```
Comment (thailekha) [inline test/unit/test-static-route.js:25]: `cloud` is not in https://github.com/feedhenry-templates/appforms-datasources-mongodb/blob/master/application.js ?

File: test/unit/test-static-route.js [hunk 2]
```diff
@@ -1,14 +1,67 @@
...
+        return function (req, res, next) { return next(); };
+      },
+      errorHandler: function () {
+        return function (req, res, next) { return next(); };
+      }
+    };
+  },
+  db: function (arg1, arg2) {
```
Comment (thailekha) [inline test/unit/test-static-route.js:33]: `db` is not in https://github.com/feedhenry-templates/appforms-datasources-mongodb/blob/master/application.js ?

File: test/unit/test-static-route.js [hunk 3]
```diff
@@ -1,14 +1,67 @@
...
-exports['test returned months data'] = function (done) {
-  request(app)
-  .get('/assets').expect(200).end(function (err, response) {
-    var body = response.body;
-    assert(Array.isArray(body), 'response body should be an array');
-    assert.equal(body[0].key, 0, 'first key is 0');
-    assert.equal(body[0].value, 'Office Chairs', 'first value is Office Chairs');
-    return done();
-  });
+var corsMock = function () {
```
Comment (sedroche) [inline test/unit/test-static-route.js:5]: Is this used?

Comment (thailekha) [inline test/unit/test-static-route.js:5]: Is this used anywhere?

**Pull request merged**